src/opm/output/eclipse/CreateDoubHead.cpp

```diff
@@ -140,10 +140,15 @@ namespace {
             if (sched_state.network().active()) {
                 const auto lookup_step = report_step - 1;
                 balancingInterval = units.from_si(M::time, sched[lookup_step].network_balance().interval());
-                convTolNodPres = units.from_si(M::pressure, sched[lookup_step].network_balance().pressure_tolerance());
-                convTolTHPCalc = sched[lookup_step].network_balance().thp_tolerance();
-                targBranchBalError = units.from_si(M::pressure, sched[lookup_step].network_balance().target_balance_error());
-                maxBranchBalError = units.from_si(M::pressure, sched[lookup_step].network_balance().max_balance_error());
+                convTolNodPres = (sched[lookup_step].network_balance().pressure_tolerance() > 0.) ?
```

Comment:
Since all of these line access the `sched[..].network_balance()` I would suggest creating a local variable - i.e. something like:
```C++
const auto& nb = sched[lookup_step].network_balance();
convTolNodes = (nb.pressure_tolerance > 0) ? units.from_si(nb.pressure_tolerance()) : units.from_si(M::pressure, 1E4);
...
```